src/main/java/org/folio/circulation/resources/PickSlipsResource.java

```diff
@@ -98,7 +101,7 @@ private void getMany(RoutingContext routingContext) {
     UUID servicePointId, Clients clients) {
 
     return findWithCqlQuery(clients.locationsStorage(), LOCATIONS_KEY, Location::from)
-      .findByQuery(exactMatch(PRIMARY_SERVICE_POINT_KEY, servicePointId.toString()));
+      .findByQuery(exactMatch(PRIMARY_SERVICE_POINT_KEY, servicePointId.toString()), LOCATIONS_LIMIT);
```

Comment:
Is this really a locations limit now that it's also used for service points as well?